src/main/java/io/github/dsheirer/module/decode/p25/P25TrafficChannelManager.java

```diff
@@ -309,6 +309,8 @@ private void processPhase1ChannelGrant(APCO25Channel apco25Channel, ServiceOptio
                     broadcast(event);
                     SourceConfigTuner sourceConfig = new SourceConfigTuner();
                     sourceConfig.setFrequency(frequency);
+                    if(mParentChannel.getSourceConfiguration() instanceof  SourceConfigTuner parentConfigTuner)
+                        sourceConfig.setPreferredTuner(parentConfigTuner.getPreferredTuner());
```

Comment:
Could you please wrap each of the sub-clause additions in braces.  I know Java allows single-line sub-clause without braces, but it can lead to logic/readability errors.